latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          dec:
+            title: Declination (deg)
+            description: |
+              Characteristic declination in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          s_region:
+            title: STC-S formatted sky footprint
+            description: |
+              Sky footprint for this data product, formatted as an
+              IVOA Space-Time Coordinate Metadata Linear String
+              (STC-S) in units of degrees.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [reference_frame, ra, dec, s_region]
+      pixel_scale:
+        # Optional
+        title: Pixel scale of spatial axes (arcsec/pixel)
+        type: number
+        unit: "arcsec/pixel"
+      spatial_dimensions:
+        # Optional
+        title: Pixel dimensions of spatial axes
+        description: |
+          Pixel dimensions of spatial axes for this data product. Used
+          to improve archival searches in the Virtual Observatory
+          (VO).
+        type: object
+        properties:
+          naxis1:
+            # Optional
+            title: Number of pixels along spatial axis 1
+            type: integer
+          naxis2:
+            # Optional
+            title: Number of pixels along spatial axis 2
+            type: integer
+      waveband:
+        title: Waveband
+        description: |
+          Waveband of this data product, used to enable archival
+          searches. Select from: RADIO, MILLIMETER, INFRARED, OPTICAL,
+          UV, EUV, XRAY, GAMMARAY.
+        anyOf:
+          - type: "null"
+          - type: string
+          - type: array
+            items:
+              type: string
+      wave_min:
+        title: Minimum wavelength (units in documentation)
+        description: |
+          Minimum wavelength bound of this data product, used to
+          enable archival searches. Specify units in MAST documentation.
+        anyOf:
+          - type: number
+          - type: "null"
+      wave_max:
+        title: Maximum wavelength (units in documentation)
+        description: |
+          Maximum wavelength bound of this data product, used to
+          enable archival searches. Specify units in MAST documentation.
+        anyOf:
+          - type: number
+          - type: "null"
+      source_observations_id:
+        # Optional
+        title: Roman observations from which this product was derived
+        description: |
+          Roman observation identifier(s) (PPPPPCCAAASSSOOOVVVggsaaeee)
```

```suggestion
          Roman observation identifier(s) (PPPPPCCAAASSSOOOVVVggsaaeee)
```
Does this correspond to `observation.observation_id`?
https://github.com/spacetelescope/rad/blob/7f58522372d07ad1fade00c9ee7445a912ffa660/latest/meta/observation.yaml#L13C11-L13C38
If so can we standardize and use that name?

---

Well, observation_id is the Roman observation identifier of the product. This is the Roman observation identifier(s) for the products from which this product was derived. IMO different things? (Also, for that reason, this can be an array.)

---

Thanks. I'll leave a new suggestion to hopefully clarify this.